docs/manuals/source/TasksAndConcepts/CatalogMaintenance.rst

```diff
@@ -681,3 +714,56 @@ Assuming that you start all your nightly backup jobs at 1:05 am (and that they r
    }
 
 It is preferable to write/send the :ref:`bootstrap <BootstrapChapter>` file to another computer. It will allow you to quickly recover the database backup should that be necessary. If you do not have a bootstrap file, it is still possible to recover your database backup, but it will be more work and take longer.
+
+
```

Comment:
Is using the PGDG Packages really what we  recommend?
Why are people paying for the OS like RHEL where they get more recent postgres versions from their OS vendor?

Reply:
Because you don't want to run 9.2 or 9.6 on EL7 for example.
Because you have a beefy server and you will be better serve by a 15 with // process than the default 12 or 13 given by RHEL.

At least now rhel as start to move a bit forward, so people will have less and less reason to move to pgdg, but still there will be some case. If they use them, then we should just explain how to succeed on first run with bareos.

Reply:
Ok re-wording done.